libs/market-list/src/lib/use-market.ts

```diff
@@ -0,0 +1,88 @@
+import { useYesterday } from '@vegaprotocol/react-helpers';
+import * as Schema from '@vegaprotocol/types';
+import isEqual from 'lodash/isEqual';
+import { useEffect } from 'react';
+import {
+  MarketsDataHookDocument,
+  useMarketHookQuery,
+} from './__generated__/MarketsHooks';
+import type {
+  MarketHookDataFragment,
+  MarketHookQuery,
+  MarketsDataHookSubscription,
+  MarketsDataHookSubscriptionVariables,
+} from './__generated__/MarketsHooks';
+
+export type MarketX = NonNullable<MarketHookQuery['market']>;
```

Comment:
can we such arbitrarily assume here _market_ in results of query will never be null or undefined?